roles/kube_vip/tasks/main.yml

```diff
@@ -40,13 +40,23 @@
   notify:
     - Restart "kubelet" service
 
-- name: Upload Kubernetes manifest
+- name: Upload Kubernetes manifest (if ARP for VIP)
   ansible.builtin.template:
     src: kube-vip.yaml.j2
     dest: /etc/kubernetes/manifests/kube-vip.yaml
     owner: root
     group: root
     mode: "0644"
+  when: not kube_vip_bgp_enabled
+
+- name: Upload Kubernetes manifest (if BGP for VIP)
+  ansible.builtin.template:
+    src: kube-vip.yaml.j2
```

Comment:
Would this need to be kube-vip-bgp.yaml.j2?

Reply:
thank you . fixed.